### DISCUSSION TITLE
When we will be able to select from multiple tables in ORM?

### DISCUSSION THREAD

**Discussion by B-Esmaili** [vlang/v#16757] [Questions and Answers] [answered]
Right now we can only select from one table in a ORM query , as selection from multiple tables is a most-have feature, I wonder when does this feature will be implemented? will it make it to 0.4 release?

**Comment by enghitalo** [comment, marked as answer]
> Right now we can only select from one table in a ORM query , as selection from multiple tables is a most-have feature, I wonder when does this feature will be implemented? will it make it to 0.4 release?

This feature is not so hard to be implemented, today. PR are well welcome. ❣️

**Reply by B-Esmaili** [reply]
Frankly I am so eager to contribute to the project , but I am not sure if I have enough skill in the language to do thing right. But I'll do my best.

**Reply by enghitalo** [reply]
Maybe, this PR can help you, as example: https://github.com/vlang/v/pull/15693

**Reply by B-Esmaili** [reply]
I am not good at c. The `vlib/v/gen/c/sql.v` file looked intimidating for me. I have no idea what's exactly going on in this file.
